.storybook/preview-head.html

```diff
@@ -4,6 +4,7 @@
     href="https://fonts.googleapis.com/css?family=Inconsolata"
 />
 <link href="https://fonts.googleapis.com/css2?family=Source+Serif+Pro&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="./sb-styles/vars.css" />
```

Comment:
**note:** I fixed this import that was not working properly in the Storybook instance.

Reply:
Huh... that's surprising that the imports weren't working properly?

Reply:
Yeah, for some reason Storybook doesn't resolve properly when we try to use `@import` inside CSS files (seems to be a problem with relative paths).